### ISSUE TITLE
wrong distfile size

### ISSUE THREAD

**Issue by zen2** [onkelbeh/HomeAssistantRepository#10]
```
>>> Downloading 'https://github.com/home-assistant/home-assistant/archive/0.106.2.tar.gz'
--2020-03-03 13:14:46--  https://github.com/home-assistant/home-assistant/archive/0.106.2.tar.gz
Resolving github.com... 140.82.118.4
Connecting to github.com|140.82.118.4|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://github.com/home-assistant/core/archive/0.106.2.tar.gz [following]
--2020-03-03 13:14:47--  https://github.com/home-assistant/core/archive/0.106.2.tar.gz
Reusing existing connection to github.com:443.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/home-assistant/core/tar.gz/0.106.2 [following]
--2020-03-03 13:14:47--  https://codeload.github.com/home-assistant/core/tar.gz/0.106.2
Resolving codeload.github.com... 140.82.114.10
Connecting to codeload.github.com|140.82.114.10|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-gzip]
Saving to: ‘/var/cache/distfiles/homeassistant-0.106.2.tar.gz.__download__’

/var/cache/distfiles/homeassistant-     [               <=>                                               ]   5.01M  1.61MB/s    in 3.1s    

2020-03-03 13:14:50 (1.61 MB/s) - ‘/var/cache/distfiles/homeassistant-0.106.2.tar.gz.__download__’ saved [5255006]

!!! Fetched file: homeassistant-0.106.2.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      5255006
!!! Expected: 5255766
Refetching... File renamed to '/var/cache/distfiles/homeassistant-0.106.2.tar.gz._checksum_failure_.e812oi1b'

!!! Couldn't download 'homeassistant-0.106.2.tar.gz'. Aborting.
 * Fetch failed for 'app-misc/homeassistant-0.106.2'
```

If I force to regenerate manifest I got:
```
>>> Emerging (1 of 1) app-misc/homeassistant-0.106.2::HomeAssistantRepository
 * homeassistant-0.106.2.tar.gz BLAKE2B SHA512 size ;-) ...                                                                           [ ok ]
>>> Unpacking source...
>>> Unpacking homeassistant-0.106.2.tar.gz to /var/tmp/portage/app-misc/homeassistant-0.106.2/work
>>> Source unpacked in /var/tmp/portage/app-misc/homeassistant-0.106.2/work
 * ERROR: app-misc/homeassistant-0.106.2::HomeAssistantRepository failed (prepare phase):
 *   The source directory '/var/tmp/portage/app-misc/homeassistant-0.106.2/work/home-assistant-0.106.2' doesn't exist
 * 
 * Call stack:
 *            ebuild.sh, line  762:  Called __ebuild_main 'prepare'
 *   phase-functions.sh, line 1054:  Called __dyn_prepare
 *   phase-functions.sh, line  384:  Called die
 * The specific snippet of code:
 *   		die "The source directory '${S}' doesn't exist"
 * 
 * If you need support, post the output of `emerge --info '=app-misc/homeassistant-0.106.2::HomeAssistantRepository'`,
 * the complete build log and the output of `emerge -pqv '=app-misc/homeassistant-0.106.2::HomeAssistantRepository'`.
 * The complete build log is located at '/var/tmp/portage/app-misc/homeassistant-0.106.2/temp/build.log.gz'.
 * The ebuild environment file is located at '/var/tmp/portage/app-misc/homeassistant-0.106.2/temp/environment'.
 * Working directory: '/var/tmp/portage/app-misc/homeassistant-0.106.2/homedir'
 * S: '/var/tmp/portage/app-misc/homeassistant-0.106.2/work/home-assistant-0.106.2'
```

**Comment by onkelbeh** [comment]
Yeah, these guys renamed the main repository from `'homeassistant'` to `'core'` some days ago. Furthermore they redirect the old archives and chages the source folder name. Tsss...

Should be fixed in the current release. I did not test all versions, but 0.106.4 & 0.106.5 compile fine.
You can keep the old distfiles, I renamed them to `homeassistant-core-${PV}.tar.gz`.

**Comment by onkelbeh** [comment]
Fixed in [update homeassistant-0.106.[3-5], new repo name](https://git.edevau.net/onkelbeh/HomeAssistantRepository/commit/03ce0829ee0adf8c86f405df6cd193262623ba90).
